app/invocation/invocation_suggestion_card.tsx

```diff
@@ -70,6 +71,108 @@ const matchers: SuggestionMatcher[] = [
       </>
     ),
   }),
+  // Suggest remote.buildbuddy.io instead of cloud.buildbuddy.io
+  ({ model }) => {
+    if (!capabilities.config.expandedSuggestionsEnabled) return null;
+
+    // remote.buildbuddy.io doesn't support cert-based auth
+    if (model.optionsMap.get("tls_client_certificate")) return null;
+    const flags = [];
+    for (const flag of ["bes_backend", "remote_cache", "remote_executor"]) {
+      const value = model.optionsMap.get(flag) || "";
+      if (value === "cloud.buildbuddy.io" || value === "grpcs://cloud.buildbuddy.io") {
+        flags.push(flag);
+      }
+    }
+    if (!flags.length) return null;
+    return {
+      level: SuggestionLevel.INFO,
+      message: (
+        <>
+          This build is using the old BuildBuddy endpoint, <span className="inline-code">cloud.buildbuddy.io</span>.
+          Migrate to <span className="inline-code">remote.buildbuddy.io</span> for improved performance.
```

Comment:
@siggisim I wasn't sure if we wanted to add more details here about why it's faster, wdyt?

Reply:
I think this is fine!